python/paddle/fluid/async_executor.py

```diff
@@ -77,6 +78,17 @@ class AsyncExecutor(object):
     """
 
     def __init__(self, place=None, run_mode=""):
+        """
+        Init.
+
+        Example:
+            >>> place = fluid.CPUPlace()
+            >>> async_executor = fluid.AsyncExecutor(place)
+
+        Args:
+            place(Place): CPUPlace or GPUPlace.
```

Comment:
Seems that we do no support CUDAPlace yet.

Reply:
fixed.